packages/EasyCodingStandard/src/Console/Application.php

```diff
@@ -2,12 +2,20 @@
 
 namespace Symplify\EasyCodingStandard\Console;
 
+use Jean85\PrettyVersions;
 use Symfony\Component\Console\Application as SymfonyApplication;
 use Symfony\Component\Console\Input\InputDefinition;
 use Symfony\Component\Console\Input\InputOption;
 
 final class Application extends SymfonyApplication
 {
+    public function __construct()
+    {
+        $version = PrettyVersions::getVersion('symplify/easy-coding-standard');
+
+        parent::__construct('EasyCodingStandard', $version->getPrettyVersion());
```

Comment:
Inconsistent with Statie

Reply:
I had to do this here due to exceeded configured max line length. Do you really need to do such nitpicking?

Reply:
I can handle this, if it's too much for you. No problem.

It's just these inconsitencies lead to confusing code for following contributors. There is nothing worse than [doing 1 thing 2 ways](https://xkcd.com/927/).